### ISSUE TITLE
PDA lights should be nerfed

### ISSUE THREAD

**Issue by UbaserB** [space-wizards/space-station-14#33388]
## Description

Right now, PDA lights are too strong in the way that they make you feel safe and comfortable in a power outage, or similar situation. They're wide, bright, and enough to see stuff around you in pitch darkness. 
Maintenance is meant to be spooky, dangerous and hard to explore without the appropriate equipment (a flashlight,) to explore it. Flashlights right now are more 'optional' (excluding the use for security) than anything else right now, when it should be the opposite way around. By nerfing PDA lights, we can not only enhance the atmosphere of the station but also make flashlights more useful to crew.

There are two approaches to nerfing this: 
- Decrease radius and strength of the light, effectively requiring you to 'hug' walls around or walk blindly in the darkness hoping you're going the right way.
- Reducing the area of the light by making it directional, in front of you (similar to how current flashlights work right now) and as stated by another maintainer- almost making it feel like a buff in reality because of the bigger contrast between the darkness and the light.

Ultimately, I don't think this will affect gameplay too much as there's already emergency lights when power runs out, alongside semi-frequent lights in maintenance aiding your way.. but it will help.

**Comment by UbaserB** [comment]
For reference this is how strong they are right now,
you can see approximately a tile and a half from the center of the tile your player is standing on.

![image](https://github.com/user-attachments/assets/585223ce-08d1-41ed-8d04-e245672f0c43)

**Comment by ScarKy0** [comment]
please no even with PDA lights i already hit walls in maints constantly

**Comment by hyphenationc** [comment]
I disagree with nerfing the PDA light, or at least nerfing it harshly, but glare on your screen is a different problem

**Comment by ScarKy0** [comment]
> I disagree with nerfing the PDA light, or at least nerfing it harshly, but glare on your screen is a different problem

I know its a different problem but it also affects visibility. If you have a problem with lights reflecting then this makes the PDA light even worse, so I think the possibility should be taken into consideration.

**Comment by Everturning** [comment]
if your PDA gets stolen you can immediately tell by who because of how much brighter their light gets

**Comment by EthanQix** [comment]
> Flashlights right now are more 'optional' (excluding the use for security) than anything else right now, when it should be the opposite way around.

I'm a jani main and I can say this is absolute bollocks. Navigating maints with a 1-tile wide PDA light is a PITA, and you won't see that maints mine before you walk on it. Everyone I cross in maints uses a flashlight or even a flare because the PDA light is just insufficient.

This nerf is unnecessary.

**Comment by Ilya246** [comment]
the PDA light is already terrible no need to nerf it

also nerfing it will have gameplay implications as you would now be able to reasonably hide/sneak in the dark next to people walking, though this is maybe a good thing?

**Comment by SkippyMcSkips** [comment]
> For reference this is how strong they are right now, you can see approximately a tile and a half from the center of the tile your player is standing on.
> 
> ![image](https://private-user-images.githubusercontent.com/134914314/387116528-585223ce-08d1-41ed-8d04-e245672f0c43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5NTM1ODIsIm5iZiI6MTczMTk1MzI4MiwicGF0aCI6Ii8xMzQ5MTQzMTQvMzg3MTE2NTI4LTU4NTIyM2NlLTA4ZDEtNDFlZC04ZDA0LWUyNDU2NzJmMGM0My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExOFQxODA4MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NzhlZTExMzZmOTFlYTUzYTg3NzQ1YjNmYWZmZDZiYmEzYTdkYTQ0YmI4NzFiNTFmMWQ3ZDQ3N2Q4NjM5YjM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ag7r6bjT2vrI9nU9RXHnPJqZ5OdFwIwNA2e3X7uaOOY)

And a tile and a half is "too much"? What are you proposing, half a tile?

>effectively requiring you to 'hug' walls around or walk blindly in the darkness hoping you're going the right way.

Yeah that sounds very fun...

**Comment by IamVelcroboy** [comment]
Everybody up in arms as if there aren't tons of flashlights all over the station...

**Comment by thebadman4662** [comment]
> Everybody up in arms as if there aren't tons of flashlights all over the station...

If you are using PDA light then either you did your job when engi forgot to turn on AME or someone looted all the cells for something as usual. Then there are EMPs, so it would be nice to have IC way to tell if its wall in your way or some console that is not rightclicking whatever the hell is standing in your way and oh suddenly you know exactly that its unwrenched ID card console in those EMPd dark arrivals.

**Comment by Sarahon** [comment]
> Everybody up in arms as if there aren't tons of flashlights all over the station...

lmfao. There should be some utility to a pda light, otherwise you might as well just remove it. The flashlight is the answer to the crappy pda lighting, not the answer to no light. The PDA is often your only light you get most rounds.

**Comment by thebadman4662** [comment]
> > Everybody up in arms as if there aren't tons of flashlights all over the station...
> 
> lmfao. There should be some utility to a pda light, otherwise you might as well just remove it. The flashlight is the answer to the crappy pda lighting, not the answer to no light. The PDA is often the only permanent light you get most rounds.

Or light implant meta. :godmode:

**Comment by Pumkin69** [comment]
no nerfing pda light its literally fine as is and i already have to hug walls to see in maints.

**Comment by SlamBamActionman** [comment]
We unintentionally lowered the PDA's light when we did an engine lighting rework, and it made it annoyingly uncomfortable to use. Right now the light is strong enough that you can really only tell whether you are walking into a wall or not, and what's immediately in front of you on the floor. Pretty sure blindness gives a larger radius. 

With Scar's comment as maintainer I am optiong to close this as not planned.
